auth-api/migrations/versions/885632ab6357_insert_tasks_table_with_pending_orgs.py

```diff
@@ -41,6 +43,28 @@ def upgrade():
                    f"('{created_time}', '{created_time}', '{name}', '{date_submitted}', '{task_relationship_type}',"
                    f" {org_id}, {user_id}, {user_id}, {user_id}, '{status}', '{task_type}')")
 
+    # Let us seed Tasks table with the existing rejected accounts
+    org_res = conn.execute(f"SELECT * FROM orgs WHERE status_code = 'REJECTED';")
```

Comment:
You'll need a new migration file instead of reuse existing one. Database might already migrate to this version,

Reply:
Yeah, we are planning to downgrade and upgrade in DEV